spec/admin/contributions_controller.spec.rb

```diff
@@ -138,4 +138,50 @@
                   .and have_content(Contribution.human_attribute_name("state/#{contribution_to_show.state}"))
     end 
   end
+
+  describe 'GET edit' do
+    let(:contribution) { create(:contribution) }
+
+    it 'has http status 200' do
+      get :edit, params: { id: contribution.id }
+      expect(response).to have_http_status(:ok)
```

Comment:
```suggestion
      get :edit, params: { id: contribution.id }
      
      expect(response).to have_http_status(:ok)
```

we could add a new line in order to keep the style consistent across spec files

Reply:
Sure, already fixed